tests/method-chain/inline_merge.js

```diff
@@ -0,0 +1,8 @@
+Object.keys(
+  availableLocales({
+    test: true
+  })
+)
+.forEach(locale => {
```

Comment:
Shouldn't this be indented? It's a little weird that it's on the same level as `Object`. We usually indent sub-expressions to make it clearer.